src/components/Header/Header.css

```diff
@@ -1,8 +1,8 @@
 :root {
```

Comment:
Don't repeat code for CSS variables, you only need to add it in one place then use everywhere in the application.